my-school/src/Components/Dashboards/Nav/NavMenu.js

```diff
@@ -11,6 +11,13 @@ import {
 } from "@chakra-ui/core";
 
 const NavMenu = ({ user }) => {
+
+  // Logout click handler...currently removes everything in localstorage but could be updated to be more specific
+  const handleLogout = e => {
+    e.preventDefault();
+    localStorage.clear();
+  }
```

Comment:
This will probably eventually need a history push and maybe a state update rerender the nav without the logout button. Just a note for future